test/v1_0_2/configs/activities.js

```diff
@@ -571,96 +571,6 @@
                 ]
             },
             {
-                name: 'An Activity Definition uses the "interactionType" property if any of the correctResponsesPattern, choices, scale, source, target, or steps properties are used (Multiplicity, 4.1.4.1.t)',
```

Comment:
interactionType and other fields used are both still optional
https://github.com/adlnet/xAPI-Spec/blob/master/xAPI.md#interactionacts